client/src/components/_common/Button/Button.module.css

```diff
@@ -40,7 +40,7 @@
 .as-link {
   composes: c-button--as-link from '../../../styles/components/c-button.css';
   background-color: transparent;
-  border-color: transparent;
+  border-color: transparent !important;
```

Comment:
You can make this `border-color` take effect *without* using [the Death Star a.k.a. `!important`][specificity-wars].

```
button.as-link {
  ...
  border-color: transparent;
  ...
}
```

<details>

I'm not sure why the style did not apply with just a class selector…

**but** this gentler solution works because [`element.class` is stronger than `.class`][specificity-wars].

</details>

[specificity-wars]: https://stuffandnonsense.co.uk/archives/images/css-specificity-wars.png

Reply:
Garret figured out a solution that is more specific **and** makes the module more robust **and** teaches other readers what is being overwritten. 👍